style.css

```diff
@@ -12,7 +12,7 @@ html {
   background: #f7f7f9;
 }
 
-@media only screen and (min-width: 375px) {
+@media (min-width: 375px) and (max-width: 600px) {
```

Comment:
- [ ] Please remove the **(max-width: 600px)** because your design is breaking or your css styles are not working >=600px and < 768px screen size which makes your website looks unprofessional with bad user experience. Kindly have a look at the below image to see the problem on screen size 680px.

### Your Design

![image](https://github.com/Afsheen96/Portfolio/assets/66476329/9620fdd1-23d3-4265-8936-6122d328c52f)


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)
![Screenshot from 2023-06-10 13-55-49](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/66476329/e5edbd52-63c9-4a93-a82c-c7b4fe7f7026)